ambari/datadog_checks/ambari/ambari.py

```diff
@@ -2,16 +2,14 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 from datadog_checks.base import AgentCheck
-from six.moves.urllib.parse import quote, urljoin
-
 
 CLUSTERS_URL = "http://{ambari_server}:{ambari_port}/api/v1/clusters"
+HOST_URL = "http://{ambari_server}:{ambari_port}/api/v1/clusters/{cluster_name}/hosts/{host_name}"
```

Comment:
carried over previous PR, guess we can delete later if not used

Reply:
i'm using the HOSTS_URL instead specifically for metrics, but may be useful to keep this in anyway for more basic host requests